docs/source/release_notes.rst

```diff
@@ -70,6 +71,9 @@ Release Notes
     Thanks to the following people for contributing to this release:
     :user:`johnbridstrup`, :user:`jeff-hernandez`, :user:`tamargrey`, :user:`thehomebrewnerd`
 
+**Breaking Changes**
+    * Removes ``DataTable`` and has the new Accessor approach.
```

Comment:
I think we should probably add a little more detail here.

Maybe something like this:

'The `DataTable` and `DataColumn` classes have been removed and replaced by new `WoodworkTableAccessor` and `WoodworkColumnAccessor` classes which are used through the `ww` namespace available on DataFrames after importing Woodwork.'

Reply:
✅ fae3c37